python/ray/_private/worker.py

```diff
@@ -160,9 +165,9 @@ def __init__(self, function: Callable[[T0, T1], R]) -> None:
 
     def remote(
         self,
-        __arg0: "Union[T0, ObjectRef[T0]]",
-        __arg1: "Union[T1, ObjectRef[T1]]",
-    ) -> "ObjectRef[R]":
+        __arg0: "Union[T0, ObjectRefType[T0]]",
```

Comment:
This is a bug I found. When I passed object ref, mypy couldn't find the correct type, but it found ray._raylet.ObjectRef

Reply:
cc @pcmoritz